packages/eslint-plugin-leboncoin/lib/rules/prefer-rtl-namespace.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+  meta: {
+    docs: {
+      description: 'Prefer use of `rtl` namespace over `@testing-library/react` (prefer-rtl-namespace)',
+      category: 'Best practices',
+      recommended: false,
+    },
+    fixable: null,
+    schema: [],
+    type: 'problem',
+  },
+
+  create: context => ({
+    ImportDeclaration (node) {
+      const identifier = node.source.value
+
+      if (identifier !== '@testing-library/react') return
+
+      context.report(
+        node,
+        `Unexpected “${identifier}”: prefer the use of the “rtl” namespace`
```

Comment:
Awesome rule!

I would recommend (but not mandatory) to have a look at https://eslint.org/docs/developer-guide/working-with-rules#messageids to avoid repeating error message in the rule and in the test.

It requires eslint@5^ at least

Check out this example in the eslint-plugin-brikke package:
https://github.com/leboncoin/frontend-web-tools/blob/master/packages/eslint-plugin-brikke/lib/rules/no-undeclared-imports.js#L42